profile/index.php

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>VITA lincoln</title>
+	<?php
+		$root = realpath($_SERVER["DOCUMENT_ROOT"]);
+		require_once "$root/server/header.php";
+	?>
+	<link href="/assets/css/main.css" rel="stylesheet">
+	<link href="/assets/css/form.css" rel="stylesheet">
+</head>
+<body>
+	<?php
+		$page_subtitle = 'Volunteer Profile';
+		require_once "$root/components/nav.php";
+	?>
+  <div>
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col col-12 col-sm-8">
+              <h2 class="form-title" id="nameHeaderProfile"></h2>
+              <h4>Name</h4>
+              <p id="nameProfile"></p>
+              <h4>Phone Number</h4>
+              <p id="phoneProfile"></p>
+              <h4>Email</h4>
+              <a class="emailProfileStatic" href="mailto:joe@example.com?subject=feedback" id="emailProfile"></a>
+              <h4>Foreign Languages</h4>
+              <ul id="#languageSkills">
```

Comment:
I doubt it matters, but you probably shouldn't put the `#` in front of the id, that's a JQuery thing